express/api/file-utils.js

```diff
@@ -96,9 +97,28 @@ module.exports.getConfigObject = getConfigObject;
  * @param {object} configObject
  */
 async function saveConfigJson(project, configObject) {
+  try {
+    const config = decrypt(await fs.promises.readFile(`${appConfig.projectDirectory}/${project}/config/ora2pg-conf.json.enc`));
+    const saved = JSON.parse(config);
+    if (saved.COMMON.values.LAST_MODIFIED) {
```

Comment:
Could we do something like this to avoid nested ifs?
```
if ((saved.COMMON.values.LAST_MODIFIED) && 
     (!configObject.COMMON.values.LAST_MODIFIED ||
       configObject.COMMON.values.LAST_MODIFIED.value !==
       saved.COMMON.values.LAST_MODIFIED.value) {
     return false;
}
```

Reply:
also not sure if `!=` instead of `!==` was intentional

Reply:
Although I don't really like using !=. I could cast them both to numbers instead